src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTestCommon/Options/CollectionRuleOptionsExtensions.cs

```diff
@@ -55,28 +54,27 @@ public static CollectionRuleOptions AddAction(this CollectionRuleOptions options
             return options;
         }
 
-        public static CollectionRuleOptions AddCollectDumpAction(this CollectionRuleOptions options, string egress = null, DumpType? type = null)
+        public static CollectionRuleOptions AddCollectDumpAction(this CollectionRuleOptions options, string egress = null, Action<CollectDumpOptions> callback = null)
```

Comment:
I updated the extension methods in this class to consistently use a delegate for optionally configuring more settings.